ipa-core/tests/common/mod.rs

```diff
@@ -189,18 +185,39 @@ pub fn spawn_shards(
                         .args(["-i", &id.to_string()])
                         .args(["--shard-index", &shard_index.to_string()])
                         .args(["--shard-count", &shard_count.to_string()])
-                        .args(["--network".into(), config_path.join("network.toml")])
-                        .arg("--disable-https")
```

Comment:
so this is the first time we have https tests?

Reply:
for sharded world yes